Stripe/PublicHeaders/STPConnectAccountParams.h

```diff
@@ -26,16 +36,26 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, nullable, readonly) NSNumber *tosShownAndAccepted;
 
 /**
- Required property with information about the legal entity for this account.
+ The business type.
+ */
+@property (nonatomic, readonly) STPConnectAccountBusinessType businessType;
 
- At least one field in the legalEntity must have a value, otherwise the create token
- call will fail.
+/**
+ Information about the individual represented by the account.
+ 
+ @see https://stripe.com/docs/api/tokens/create_account#create_account_token-account-individual
+ */
+@property (nonatomic, nullable, readonly) NSDictionary *individual;
```

Comment:
why are these dictionaries instead of objects?

Reply:
Only because it's a lot of parameters and Android's API looks like this.

Reply:
hmm, I think we should type these. https://github.com/stripe/stripe-ios/pull/1178 has this but I haven't reviewed it extensively